include/soundanalyzer.h

```diff
@@ -415,7 +417,8 @@ class SoundAnalyzer : public AudioVariables
         // We'll use the average as the gVU.  I assume the average of the samples tracks sound pressure level, but don't really know...
 
         float newval = averageSum / (MAX_SAMPLES / 2 - 2);
-        debugV("AverageSumL: %f", averageSum);
+        debugV("AverageSum : %f", averageSum);

```

Comment:
I don't think these should be Verbose, not sure how to correct once we're in a review?

Reply:
Actually I had it backwards, I DO want these to be verbose only.  I was thinking backwards!

Reply:
Well in that case it's even easier: you should **not** push a change to davepl:main. :smile: